app/Models/Reviews.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Reviews extends Model
```

Comment:
I think the many-to-many linking tables need to extend the pivot class based on what I saw here: https://laravel.com/docs/8.x/eloquent-relationships#defining-custom-intermediate-table-models